tf_existing_net_rds_ec_ec2/securityGroups.tf

```diff
@@ -135,26 +135,56 @@ resource "aws_security_group" "bastion_instance" {
   vpc_id      = var.vpc_id
 
   ingress {
-    from_port       = 22
-    to_port         = 22
-    protocol        = "tcp"
-    cidr_blocks     = split(",", var.ingress_ips)
-    description     = var.ingress_ip_description
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = split(",", var.ingress_ips)
+    description = var.ingress_ip_description
   }
 
   egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name          = "${var.prefix}_bastion"
+    application   = var.tag_application
+    contact-email = var.tag_contact_email
+    customer      = var.tag_customer
+    team          = var.tag_team
+    environment   = var.tag_environment
+  }
+}
+
+# EFS Security Group
+resource "aws_security_group" "airflow_efs" {
+  name        = "${var.prefix}_efs"
+  description = "Security group for access to efs mounts for airflow"
+  vpc_id      = var.vpc_id
+
+  ingress {
     from_port       = 0
     to_port         = 0
     protocol        = "-1"
-    cidr_blocks     = ["0.0.0.0/0"]
+    security_groups = ["${aws_security_group.airflow_instance.id}"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
   }
 
   tags = {
-    Name            = "${var.prefix}_bastion"
-    application     = var.tag_application
-    contact-email   = var.tag_contact_email
-    customer        = var.tag_customer
-    team            = var.tag_team
-    environment     = var.tag_environment
+    Name          = "${var.prefix}_efs"
```

Comment:
Do we still need the `_bastion` security group?

Reply:
Yes.  You will need this.